src/apps/profiles/models.py

```diff
@@ -38,3 +52,43 @@ def get_full_name(self):
 
     def __str__(self):
         return self.name if self.name else self.username
+
+    @property
+    def chahub_uid(self):
+        associations = self.social_auth.filter(provider='chahub')
+        if associations.count() > 0:
+            return associations.first().uid
+        return None
+
+    def get_chahub_endpoint(self):
+        return "profiles/"
+
+    def clean_chahub_data(self, temp_data):
+        data = temp_data
+        for key in list(data):
```

Comment:
Would it make more sense to only _add_ what we want to data, rather than add everything to data and try and clean it? i.e.
```python
data = {}
for key, item in temp_data.items():
    if not item or key in PROFILE_DATA_BLACKLIST:
        continue
    elif key == 'details':
        item = self.clean_chahub_data(item)
    elif isinstance(item, datetime.datetime):
        item = item.isoformat()
    data[key] = item
```